### PR TITLE
task3.5 added ready for review

### PR DESCRIPTION
**Pull request by Kylehinks97**
About to put in a /nchelp review as well :)

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return connection.end();
+});
+
+describe.skip("TASK 3 -- api/topics", () => {
```
Comment (cjplatten) [inline __tests__/app.test.js]: Delete any .skips or .onlys before commiting code.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+describe("TASK 3.5 --- GET - /api", () => {
+  it("GET - status: 2 - responds with JSON object containing all the endpoints in endpoints.test.js", () => {
+    return request(app)
+      .get("/api")
+      .then((response) => {
+        expect(response.statusCode).toBe(200);
+        expect(typeof response).toBe("object");
+      });
+  });
+  it("GET - status: 2 - responds with JSON object containing all the endpoints in endpoints.test.js", () => {
```
Comment (cjplatten) [inline __tests__/app.test.js]: you only need one 404 for incorrect urls for the whole as that is its own error.

File: db/controllers.js [hunk 3]
```diff
@@ -1,14 +1,28 @@
-const { selectTopics } = require("./models.js")
+//models
+const { selectTopics } = require("./models.js");
+const fs = require("fs");
 
 exports.getTopics = (req, res) => {
-    console.log("in controller");
-    selectTopics()
+  console.log("in controller");
```
Comment (cjplatten) [inline db/controllers.js]: remove any console.logs

File: db/controllers.js [hunk 4]
```diff
@@ -1,14 +1,28 @@
...
+  selectTopics()
     .then((result) => {
-        console.log(result.rows);
-        res.status(200).send({ topics: result.rows })
+      console.log(result.rows);
```
Comment (cjplatten) [inline db/controllers.js]: console.log!

**Review by cjplatten — COMMENTED** [review]
Please see comments. 
Be careful when branching off to make new changes and when making commits, there are a lot of changes in this ticket that are not for this ticket and are for ticket 3 instead.

File: __tests__/app.test.js (hunk 1 above)
Comment (Kylehinks97) [reply __tests__/app.test.js]: I have made the changes and pushed, do I make a new PR? or?

**Review by Slyose — APPROVED** [review]
Looks good! ✅ 

*after you get rid of the second 404 test

**Pull request merged**